operations/kubernetes/Troubleshoot_Postgres_Database.md

```diff
@@ -175,8 +175,87 @@ postgres@keycloak-postgres-2:~$ patronictl list
 |      Cluster      |        Member       |     Host     |  Role  |  State  | TL | Lag in MB |
 +-------------------+---------------------+--------------+--------+---------+----+-----------+
 | keycloak-postgres | keycloak-postgres-0 | 10.42.10.22  |        | running | 47 |         0 |
-| keycloak-postgres | keycloak-postgres-1 | 10.40.11.191 | Leader | running | 47 |         0 |
-| keycloak-postgres | keycloak-postgres-2 | 10.40.11.190 |        | running | 47 |           |
+| keycloak-postgres | keycloak-postgres-1 | 10.40.11.191 | Leader | running | 47 |           |
+| keycloak-postgres | keycloak-postgres-2 | 10.40.11.190 |        | running | 47 |         0 |
++-------------------+---------------------+--------------+--------+---------+----+-----------+
+```
+
+If a cluster member is `stopped` after a successful reinitialization, check for pg_internal.init.* files that may need to be cleaned up. This can occur if the pgdata disk was full prior to the reinitialization, leaving truncated pg_internal.init.* files in the pgdata directory.
+
+```bash
+ncn-w001# kubectl exec keycloak-postgres-0 -c postgres -n services -it -- bash
+```
+
+Example output:
+
+```
```

Comment:
Is this supposed to be the output of running `kubectl exec keycloak-postgres-0 -c postgres -n services -it -- bash`? If so that would just be a prompt not this.

Reply:
Fixed